css/index.css

```diff
@@ -370,20 +370,87 @@
   border-color: #000;
 }
 
-#contact{
-  overflow: hidden;
+@media (min-width: 769px) {
+  #contact{
+    border-style: solid;
```

Comment:
I don't think this rounded outline fits in with current design on the homepage. It will be better to do away with the border altogether.